node/src/chain_spec.rs

```diff
@@ -124,101 +89,95 @@ pub fn development_config() -> Result<ChainSpec, String> {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "CAN".into());
+	properties.insert("tokenDecimals".into(), 12.into());
 
-	Ok(ChainSpec::from_genesis(
+	ChainSpec::from_genesis(
+		// Name
 		"Canvas Testnet",
-		"canvas_testnet2",
-		ChainType::Live,
+		// ID
+		"canvas_testnet",
+		ChainType::Local,
 		move || {
+            // TODO: Right now these are being left as default values, but in the old `canvas-node`
+            // we had some non-default authorities, as well as accounts for the faucent and
+            // `ink-waterfall`. We'll need to add those back in
 			testnet_genesis(
-				wasm_binary,
-				testnet_authorities(),
-				testnet_root(),
+				get_account_id_from_seed::<sr25519::Public>("Alice"),
+				vec![get_from_seed::<AuraId>("Alice"), get_from_seed::<AuraId>("Bob")],
 				vec![
-					testnet_root(),
-					// AccountId of the Canvas Testnet faucet
-					hex!("18c64aa111a8a0e6e4eed41d6d906c7614d745e48be3cfc13b6128e1d51f4405").into(),
-					// AccountId of an account which `ink-waterfall` uses for automated testing
-					hex!("0e47e2344d523c3cc5c34394b0d58b9a4200e813a038e6c5a6163cc07d70b069").into(),
+					get_account_id_from_seed::<sr25519::Public>("Alice"),
+					get_account_id_from_seed::<sr25519::Public>("Bob"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie"),
+					get_account_id_from_seed::<sr25519::Public>("Dave"),
+					get_account_id_from_seed::<sr25519::Public>("Eve"),
+					get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
 				],
-				true,
+				id,
 			)
 		},
-		// Bootnodes
-		vec![
-			"/ip4/34.90.191.237/tcp/30333/p2p/12D3KooWKg3Rpxcr9oJ8n6khoxpGKWztCZydtUZk2cojHqnfLrpj"
-				.parse()
-				.expect("MultiaddrWithPeerId"),
-			"/ip4/35.204.68.28/tcp/30333/p2p/12D3KooWPEXYrz8tHU3nDtPoPw4V7ou5dzMEWSTuUj7vaWiYVAVh"
-				.parse()
-				.expect("MultiaddrWithPeerId"),
-			"/ip4/34.90.139.15/tcp/30333/p2p/12D3KooWEVU8AFNary4nP4qEnEcwJaRuy59Wefekzdu9pKbnVEhk"
-				.parse()
-				.expect("MultiaddrWithPeerId"),
-			"/ip4/35.204.99.97/tcp/30333/p2p/12D3KooWP6pV3ZmcXzGDjv8ZMgA6nZxfAKDxSz4VNiLx6vVCQgJX"
-				.parse()
-				.expect("MultiaddrWithPeerId"),
-		],
-		// Telemetry
+        // Bootnodes
+        // TODO: We'll need some new bootnotes since the old ones from
+        //  the standalone `canvas-node` network won't work anymore
+		Vec::new(),
+        // Telemetry
 		None,
-		// Protocol ID
+        // Protocol ID
+        // TODO: Should this be `rpc`?
 		Some("prc"),
```

Comment:
I never understood why `canvas-node` uses "rpc" as the protocol id, according to [the docs on the chain spec](https://substrate.dev/docs/en/knowledgebase/integrate/chain-spec):

>nodes will only add peers who use the same `protocolId`.

And from [the rustdoc of `ProtocolId`](https://substrate.dev/rustdocs/latest/sc_network/config/struct.ProtocolId.html):

>Name of a protocol, transmitted on the wire. Should be unique for each chain. 

So I think for our rococo chain spec we should make it something like `Some("canvas-rococo")`.